docs/RouterTutorial/react-navigation-tab-kr.md

```diff
@@ -0,0 +1,411 @@
+
+   <img src="./images/04_1.png" style="zoom:25%;" />
+
+   <img src="./images/04_2.png" style="zoom:25%;" />
+
+   <img src="./images/04_3.png" style="zoom:25%;" />
+
+   
+
+   Home, Chat, Settings 3가지 화면에 대한 코드입니다.
+
+   *HomeScreen.js*
+
+   ```jsx
+   import React from 'react';
+   import {StyleSheet, Text, View} from 'react-native';
+   
+   class HomeScreen extends React.Component {
+     render() {
+       return (
+         <View style={styles.container}>
+           <Text>Home!</Text>
+         </View>
+       );
+     }
+   }
+   
+   const styles = StyleSheet.create({
+     container: {
+       flex: 1,
+       justifyContent: 'center',
+       alignItems: 'center',
+       backgroundColor: '#ff4081',
+     },
+   });
+   
+   export default HomeScreen;
+   ```
+
+   *ChatScreen.js*
+
+   ```jsx
+   import React from 'react';
+   import {StyleSheet, Text, View} from 'react-native';
+   
+   class ChatScreen extends React.Component {
+     render() {
+       return (
+         <View style={styles.container}>
+           <Text>Chat!</Text>
+         </View>
+       );
+     }
+   }
+   
+   const styles = StyleSheet.create({
+     container: {
+       flex: 1,
+       justifyContent: 'center',
+       alignItems: 'center',
+       backgroundColor: '#673ab7',
+     },
+   });
+   
+   export default ChatScreen;
+   ```
+
+   *SettingsScreen.js*
+
+   ```jsx
+   import React from 'react';
+   import {StyleSheet, Text, View} from 'react-native';
+   
+   class SettingsScreen extends React.Component {
+     render() {
+       return (
+         <View style={styles.container}>
+           <Text>Settings!</Text>
+         </View>
+       );
+     }
+   }
+   
+   const styles = StyleSheet.create({
+     container: {
+       flex: 1,
+       justifyContent: 'center',
+       alignItems: 'center',
+       backgroundColor: '#228B22',
+     },
+   });
+   
+   export default SettingsScreen;
+   
+   ```
+
+   App.js에서는 다음과 같이 navigator을 만들어주고 이를 navigate 할 수 있는 route들을 지정해줍니다.
+
+   *App.js*
+
+   ```jsx
+   import {createAppContainer} from 'react-navigation';
+   import {createBottomTabNavigator} from 'react-navigation-tabs';
+   import HomeScreen from './screens/HomeScreen';
+   import ChatScreen from './screens/ChatScreen';
+   import SettingsScreen from './screens/SettingsScreen';
+   
+   const TabNavigator = createBottomTabNavigator({
+     Home: {
+       screen: HomeScreen,
+     },
+     Chat: {
+       screen: ChatScreen,
+     },
+     Settings: {
+       screen: SettingsScreen,
+     },
+   });
+   
+   export default createAppContainer(TabNavigator);
+   ```
+
+   
+
+5. #### RN navigation tab에 react-native-vector-icons을 이용한 아이콘 및 뱃지 추가
+
+   <img src="./images/05_1.png"style="zoom:25%;" />
+
+   
+
+   Tab Navigator에서 defaultNavigationOptions를 사용하여 Tab bar의 아이콘을 지정할 수 있습니다. react-native-vector-icons등을 사용하여 더 예쁘게 구현할 수 있습니다.
+
+   [IonIcons]: https://ionicons.com/
```

Comment:
How about reflecting markdown link syntax? (I think it is better to read the doc.)

ex)
```
http://github.com - automatic! -> [GitHub](http://github.com)
```
http://github.com - automatic! -> [GitHub](http://github.com)

Reply:
<img width="824" alt="스크린샷 2019-09-19 오후 2 11 30" src="https://user-images.githubusercontent.com/34931911/65215156-7b278980-dae7-11e9-9f83-95ffb0defc5f.png">

(As you see, it doesn't appear in this img.)